sdk/core/azure-core/azure/core/pipeline/_base.py

```diff
@@ -42,6 +42,27 @@
 _LOGGER = logging.getLogger(__name__)
 PoliciesType = List[Union[HTTPPolicy, SansIOHTTPPolicy]]
 
+_ALLOWED_KWARGS = set([
```

Comment:
Should use_env_settings be included here as well? https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/core/azure-core/CLIENT_LIBRARY_DEVELOPER.md#proxy-settings